src/main/java/installer/Installer.java

```diff
@@ -1,4 +1,5 @@
 package installer;
+//TODO: does this need to be a separate package?
```

Comment:
Can all that code be put in a separate updater package and take it out of addressbook? i.e. `hubturbo.updater`